Sources/MapboxSearch/PublicAPI/MapboxSearchVersion.swift

```diff
@@ -1,2 +1,2 @@
 /// Mapbox Search SDK version variable
-public let mapboxSearchSDKVersion = "1.0.0-beta.22"
+public let mapboxSearchSDKVersion = "1.0.0-beta.23"
```

Comment:
Sorry if that's a stupid question (I'm not an iOS engineer), but why this version is not the latest, i.e. `1.0.0-beta.24`?

Reply:
This tag will be set on CI during release and new version will be released with the correct version.
I do plan to commit these changes directly from CI during release or manually, now it is a small release artefact :)